go.mod

```
@@ -1,81 +1,89 @@
 module github.com/tendermint/tendermint
 
-go 1.21
+go 1.21.0
```

```suggestion
go 1.21.13
```

why not the latest minor release?

---

@melekes Oh, probably missed that, just updated :+1: